test/e2e/manifest/testSecurityPolicy/rule-in-rule-applied-to-with-src.yaml

```diff
@@ -35,7 +35,7 @@ spec:
       hostname: web-deployment
       containers:
         - name: web
-          image: "harbor-repo.vmware.com/dockerhub-proxy-cache/humanux/http_https_echo:latest"
+          image: "netfvt-docker-local.packages.vcfd.broadcom.net/humanux/http_https_echo:latest"
```

Comment:
This is another reason I don't like to use manifest in e2e test, we need to modify many files any time the image is updated

Reply:
ok